Small changes meetingbot

emmakodes opened the pull request:
- added `Meeting_name_summary_and_attendees` to get structured output values
- added `check bot status` method
- added `_generate_meeting_summary`, `_fetch_meeting_data`, `clean_markdown` method
- now generating and saving summary to index in process_webhook method
- `get_meeting_info` method now search_collection

drbrady8800 commented on src/agentic/tools/meeting_tool.py:
```diff
@@ -1,19 +1,21 @@
-from typing import Any, Callable, Optional, Dict
+from typing import Any, Callable, Optional, Dict, List
```
```suggestion
from typing import Callable, Optional, List
```
unused imports

drbrady8800 commented on src/agentic/tools/meeting_tool.py:
```diff
@@ -24,6 +26,11 @@
 )  
 logger = logging.getLogger(__name__)
 
+class Meeting_name_summary_and_attendees(BaseModel):
```
```suggestion
class MeetingSummary(BaseModel):
```

drbrady8800 reviewed (approved):
I didn't test locally, just assume it is working. Could you make a video showing it recording your meeting and then summarizing it with you asking questions?

emmakodes commented:
Hi @drbrady8800 I renamed `Meeting_name_summary_and_attendees` to `MeetingSummary` and removed the unnecessary import. The following is the video showing recording and asking questions:

https://github.com/user-attachments/assets/1bef705b-9096-4846-b519-4fc94ebbe7e5
